pyOCD/utility/conversion.py

```diff
@@ -64,18 +64,7 @@ def float2int(data):
 
 ## @brief create 8-digit hexadecimal string from 32-bit register value.
 def intToHex8(val):
-    val = hex(int(val))[2:]
-    size = len(val)
-    r = ''
-    for i in range(8-size):
-        r += '0'
-    r += str(val)
-
-    resp = ''
-    for i in range(4):
-        resp += r[8 - 2*i - 2: 8 - 2*i]
-
-    return resp
+    return "%08X" % int(val)
```

Comment:
I think this is incorrect. When I use the original implementation, I see this:
<pre>
In [3]: intToHex8(0x01020304)
Out[3]: '04030201'
</pre>

It's reversing the byte order of the output string.

While we're fixing names here, we should probably rename these to `intToHex8LE()` and similar for the related functions.